src/com/modo/MainJava13.java

```diff
@@ -16,11 +16,12 @@ public static void main(String[] args) throws IOException, InterruptedException
         String apiKey = "API_KEY_HERE";
         String apiSecret = "API_SECRET_HERE";
 
-        String postApiUri = "/v2/reports";
-        String requestBody = "{\"start_date\": \"2020-07-13T00:00:00Z\",\"end_date\": \"2020-07-13T23:59:59Z\"}";
+        String postApiUri = "/v3/checkout/list";
+        String requestBody = "{\"checkout_ids\": []}";
 
         Modo2Auth auth = new Modo2Auth(apiKey, apiSecret);
         String authToken = auth.createModoToken(postApiUri, requestBody);
+        System.out.println("MODO2 auth token: " + authToken);
```

Comment:
Not a fan of these println statements in examples. Anyone using these examples can add their own, but considering this code is going to be copied and pasted into production work I think it would be best if the System.out.println statements are omitted.